rest/model/patch.go

```diff
@@ -166,10 +172,42 @@ func (apiPatch *APIPatch) BuildFromService(h interface{}) error {
 	apiPatch.PatchedConfig = utility.ToStringPtr(v.PatchedConfig)
 	apiPatch.Project = utility.ToStringPtr(v.Project)
 	apiPatch.CanEnqueueToCommitQueue = v.HasValidGitInfo()
+	downstreamTasks, err := getDownstreamTasks(v)
+	if err != nil {
+		return errors.Wrap(err, "error getting downstream tasks")
+	}
+	apiPatch.DownstreamTasks = downstreamTasks
 
 	return errors.WithStack(apiPatch.GithubPatchData.BuildFromService(v.GithubPatchData))
 }
 
+func getDownstreamTasks(p patch.Patch) ([]DownstreamTasks, error) {
+	downstreamTasks := []DownstreamTasks{}
+	for _, childPatch := range p.Triggers.ChildPatches {
+		childPatchDoc, err := patch.FindOneId(childPatch)
+		if err != nil {
+			return nil, errors.Wrapf(err, "error getting tasks for child patch '%s'", childPatch)
+		}
+		if childPatchDoc == nil {
+			continue
+		}
+		//set the version id to the parent id
+		childPatchDoc.Version = p.Version
```

Comment:
This wasn't what I had in mind: this doesn't seem to affect the patch we return. I was thinking about setting the [version field in the APITask](https://github.com/evergreen-ci/evergreen/blob/256e109df73a7122b5492b56acff926233203539/rest/model/task.go#L33). I was suggesting that when we set it [here](https://github.com/evergreen-ci/evergreen/blob/256e109df73a7122b5492b56acff926233203539/rest/model/task.go#L204) we case on whether the task is part of a child patch.

Reply:
resolved "in person": this will be solved in a separate ticket/PR